db/seed.sql

```diff
@@ -3723,8 +3723,8 @@ INSERT INTO `core_config_data` VALUES (356,'default',0,'tax/classes/default_prod
 INSERT INTO `core_config_data` VALUES (357,'default',0,'tax/classes/default_customer_tax_class','3');
 INSERT INTO `core_config_data` VALUES (358,'default',0,'tax/calculation/algorithm','TOTAL_BASE_CALCULATION');
 INSERT INTO `core_config_data` VALUES (359,'default',0,'tax/calculation/based_on','shipping');
-INSERT INTO `core_config_data` VALUES (360,'default',0,'tax/calculation/price_includes_tax','0');
-INSERT INTO `core_config_data` VALUES (361,'default',0,'tax/calculation/shipping_includes_tax','0');
+INSERT INTO `core_config_data` VALUES (360,'default',0,'tax/calculation/price_includes_tax','1');
+INSERT INTO `core_config_data` VALUES (361,'default',0,'tax/calculation/shipping_includes_tax','1');
```

Comment:
Client indicated they want to enter tax inclusive of GST; these two changes set that option for both products and shipping.